homeassistant/components/schlage/quality_scale.yaml

```diff
@@ -0,0 +1,84 @@
+rules:
+  # Bronze
+  config-flow: done
+  test-before-configure: done
+  unique-config-entry: done
+  config-flow-test-coverage: done
```

Comment:
`test_form_invalid_auth`, should finish with CONFIG_ENTRY so we also test that the flow is able to recover. You can combine it with `test_form_unknown` using parametrization